### PR TITLE
[MODIFY](client): clean up map functionality and add marker {CT}

### PR DESCRIPTION
**Pull request by catharp**

### CODE REVIEW

File: client/containers/Preference.jsx [hunk 1]
```diff
@@ -111,28 +112,27 @@ class Preference extends Component {
 
   render () {
     return (
-      <div className="preference">
+      <div className="container">
```
Comment (zeninspire) [inline client/containers/Preference.jsx]: question: would this interfere with the line of code in client/index.js?

document.querySelector('.container')

**Review by zeninspire — COMMENTED** [review]
in line 115 of Preference.jsx
question: would this interfere with the line of code in client/index.js?
document.querySelector('.container')

**Review by valerieernst — APPROVED** [review]
I agree with Alissa's comment RE: the div with className="container".

Other than that, looks good!

**Comment by catharp** [comment]
Oh... that!
Fixxed.

**Pull request merged**